src/bin/evtx_dump.rs

```diff
@@ -72,6 +74,12 @@ fn main() {
                 .default_value("xml")
                 .help("sets the output format"),
         )
+        .arg(
+            Arg::with_name("validate-checksums")
+                .long("--validate-checksums")
+                .takes_value(false)
+                .help("When passed, the checksum of each chunk will be validated."),
```

Comment:
We need to explain what ‘validate’ means, more so how is that different from not using this flag.